src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const func = (num) => {
```

Comment:
name variables and functions semantically, what they mean or do, not just `func`, it is not good practice. e.g. `adder`

Reply:
Done